src/api/supabase/queries/profiles.ts

```diff
@@ -13,6 +13,15 @@ export async function fetchProfiles() {
   }
 }
 
+export async function fetchProfileById(userId: UUID) {
```

Comment:
@jinkang-0 any idea why this might be here? 🤔 I assumed that this was part of the profile context PR which was already merged in...

Reply:
actually it seems like all the changes from the profile context and global styles are included on here...? i somehow thought this diff would only included changes that were not already present on main after rebasing :o but i might be misremembering!